### PR TITLE
install chromium-browser

### PR DESCRIPTION
**Pull request by alexgillcma**
- chromedriver version was out of sync with browser version
- updated setup script to install both at the same time - should resolve issue
- also a small fix to one test which was using an old selenium function

### CODE REVIEW

File: env-setup.sh [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # Install chromedriver
-apt-get update
-sudo apt install -y chromium-chromedriver
+sudo apt-get update && apt install -y chromium-chromedriver && apt install -y chromium-browser
```
Comment (dayalstrub-cma) [inline env-setup.sh:2]: can also just do 

`apt install -y chromium-chromedriver chromium-browser`

and not sure whether having separate line for `sudo apt-get update` is neater, or not.

regardless, does this fix the issue?

File: tests/test_selene_core_selenium.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def test_bool_yoffset_changed():
 def test_bool_scroll_height_changed():
     page = PageSelene.from_url(driver=driver, url = "http://www.scrapethissite.com/pages/forms/")
     orig_height = script_get_scroll_height(driver)
-    Select(driver.find_element_by_xpath('//*[@id="per_page"]')).select_by_value('100')
+    Select(driver.find_element(By.XPATH, '//*[@id="per_page"]')).select_by_value('100')
```
Comment (dayalstrub-cma) [inline tests/test_selene_core_selenium.py:63]: is this a bug fix? shall we update CHANGELOG?

Comment (alexgillcma) [reply tests/test_selene_core_selenium.py:63]: it is a bug in how one of the tests was written - it was using a now-deprecated selenium function. Not sure it needs a CHANGELOG update as it's just a test?

**Review by dayalstrub-cma — APPROVED** [review]

**Comment by dayalstrub-cma** [comment]
@alexgillcma not sure why merge is blocked.

**Pull request merged**